openlibrary/components/IdentifiersInput.vue

```diff
@@ -115,40 +114,37 @@ export default {
     },
 
     computed: {
-        popularEditionConfigs: function() {
-            if (this.edition_popular) {
-                const popularConfigs = JSON.parse(decodeURIComponent(this.edition_popular));
-                return Object.fromEntries(popularConfigs.map(e => [e.name, e]));
-            }
-            return {};
+        parsedConfig: function() {
```

Comment:
So this can then be renamed to `identifierConfig`, and we don't specify `parsed`